src/components/TracePage/KeyboardShortcutsHelp.track.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { trackEvent } from '../../utils/tracking';
+
+const context = 'jaeger/ux/trace/kbd-modal';
+
+export default function trackKbdHelpModal() {
```

Comment:
Given you're tracking an event maybe be more verbose with the event being tracked. Suggestion: `trackKbdHelpModalOpen`

Reply:
:+1: